internal/pkg/synchronization/synchronizationjob.go

```diff
@@ -0,0 +1,28 @@
+package synchronization
+
+import (
+	"github.com/PhamDuyKhang/littledetective/internal/film"
+	"github.com/PhamDuyKhang/littledetective/internal/pkg/flog"
+)
+
+func Sync(filmID chan string, elService *film.FilmService, mongoService *film.FilmMongoService) {
+	logger := flog.New()
+	logger.SetLocal("synchronization")
+	for {
+		select {
+		case idFilm, ok := <-filmID:
```

Comment:
I don't see anywhere you call this func, but again, if `filmID` never be closed then this func will never end.